ui/app/AppLayouts/Wallet/controls/NetworkFilter.qml

```diff
@@ -132,6 +134,13 @@ StatusComboBox {
                     image.source: Style.svg(model.iconUrl)
                     z: index + 1
                     visible: root.preferredNetworksMode ? root.preferredSharingNetworks.includes(model.chainId.toString()): image.source !== ""
+
+                    image.layer.enabled: index < chainRepeater.count - 1
+                    image.layer.effect: OpacityMask {
+                        maskSource: Image {
+                            source: Style.png("masks/circle-cut-0.8")
+                        }
+                    }
```

Comment:
I played a bit and I manged to eliminate external svg and make the masked icon a bit smoother on edges:

```
                        image.layer.enabled: index < chainRepeater.count - 1
                        image.layer.effect: OpacityMask {
                            invert: true

                            maskSource: Item {
                                width: ri.width
                                height: ri.height

                                Rectangle {
                                    anchors.centerIn: parent
                                    anchors.horizontalCenterOffset: 21

                                    width: parent.width + 4
                                    height: width
                                    radius: width / 2
                                }
                            }
                        }
```
Feel free to decide which option to use.

Reply:
That's a great idea! Took this chance to make it work for any spacing. Please have a look